packages/headless/src/features/commerce/configuration/configuration-state.ts

```diff
@@ -27,7 +27,6 @@ export const getConfigurationInitialState: () => ConfigurationState = () => ({
   organizationId: '',
   analytics: {
     enabled: true,
-    trackingId: '',
```

Comment:
Is this change to the commerce state needed?

Reply:
Well a trackingId being an empty string is not valid anyway.
If you send an event with it, you'll get an invalid event, so I think it's shooting yourselves in the foot to initialize it to an invalid value.

Reply:
Could the same argument be made for an empty `organizationId` on line 27? I think there are a few examples in headless where required params are set to default values, but these would never be sent because headless engine initialization requires one to specify them.

Reply:
The point is indeed the latter, that the pattern is used for options in headless that are required. Setting the default to an empty string avoids needing to use a union type with `null`, especially since `null` is not a supported value for options like `organizationId` or trackingIds in commerce